protocol/message_persistence.go

```diff
@@ -2967,9 +2979,26 @@ func (db sqlitePersistence) findStatusMessageIdsReplies(tx *sql.Tx, bridgeMessag
 	return statusMessageIDs, nil
 }
 
-// Finds status messages id which are replies for bridgeMessageID
-func (db sqlitePersistence) findStatusMessageIdsRepliedTo(tx *sql.Tx, parentMessageID string) (string, error) {
-	rows, err := tx.Query(`SELECT user_messages_id FROM bridge_messages WHERE message_id = ?`, parentMessageID)
+func (db sqlitePersistence) FindStatusMessageIdForBridgeMessageId(messageID string) (string, error) {
```

Comment:
Similar comment to above, wouldn't it be better to add some error log. I believe this can help us instrument the code in the future or just help with understanding what's going on in case there are errors. What do you think?

Reply:
ditto